WebServer.py

```diff
@@ -1,13 +1,29 @@
 from http import server
 import sys
 
+match = {
+    '/source/': '../sample/'
```

Comment:
Would it be possible for you to add a comment that explains where the "source" and "sample" folder names comes from?

If you're referring to the "sample" folder in this repo, then I would prefer to avoid hardcoding that assumption in the script, since it would make the script incompatible with other projects.

Reply:
We could define an environment variable specifying it. But it is one of the reason I wanted to put it inside the container: to avoid entering specificity of the project, these values are already hardcoded in the container.

Reply:
I would like to be able to copy&paste WebServer.py as-is into other codebases with similar need for testing WASM builds in as lightweight web-server.